GUIDE.md

```diff
@@ -2,8 +2,11 @@
 
 ## Formatting conventions
 
-These formatting conventions are a work in progress, and may do anything they
-like, up to and including eating your laundry.
+Formatting conventions are defined in the [FORMATTING.md](./FORMATTING.md) document
```

Comment:
Add a period at the end of this line.
*pedantry intensifies*

Reply:
Thanks for thoroughness :)
Fixed